bot.py

```diff
@@ -162,6 +168,123 @@ async def get_last_match(ctx, name: str):
+
+            # if server id exist in json, add number of players
+            if server_id in file_data:
+                total_number_of_players += len(file_data[server_id])
+
+            # remove summoners from incoming data if it exists in json file with same server ID
+            for player_name in player_list:
+                if any(
+                    player_name in player["user_name"]
+                    for player in file_data[server_id]
+                ):
+                    player_list.remove(player_name)
+
+        # add number of summoners from incoming data to total number of players
+        total_number_of_players += len(player_list)
+
+        if total_number_of_players > MAX_NUM_PLAYERS_TEAM:
+            raise Exception(
+                "Limit Exceeded", "You have exceeded a limit of 10 summoners!"
+            )
+
+        # make dictionary for newly coming in players
+        players_list_info = create_summoner_list(player_list, server_id)
+
+        await ctx.send(f"Total Number of Summoners: {total_number_of_players}")
+
+        # if no file exist in path or if the file is empty, dump incoming data
+        if os.path.getsize(json_path) == 0 or server_id not in file_data:
+            with open(json_path, "w") as file:
+                json.dump(players_list_info, file, indent=4)
+                file_data = players_list_info
+
+        # append data to the matching server id
+        else:
+            file_data[server_id] += players_list_info[server_id]
+
+            with open(json_path, "w") as file:
+                json.dump(file_data, file, indent=4)
+
+        embed_data = EmbedData()
+        embed_data.title = "List of Summoners"
+        embed_data.description = "** **"
+        embed_data.color = discord.Color.dark_gray()
+
+        output_str = ""
+
+        for count in range(len(file_data[server_id])):
+
+            output_str += "`{0}{1}` {2}\n".format(
+                file_data[server_id][count]["tier_division"][0],
+                TIER_RANK_MAP.get(file_data[server_id][count]["tier_rank_number"]),
+                file_data[server_id][count]["formatted_user_name"],
+            )
+
+        embed_data.fields = []
+        embed_data.fields.append(
+            {"name": "Summoners", "value": output_str, "inline": False}
+        )
+
+        await ctx.send(embed=create_embed(embed_data))
+    # pylint: disable=broad-except
+    except Exception as e_values:
+
+        if "404" in str(e_values):
+            error_title = "Invalid Summoner Name"
+            error_description = f"`{e_values.args[1]}` is not a valid name. \
+                \n\nAdding multiple summoners:\n `@{bot.user.name} add name1, name2`"
+        elif "Limit Exceeded" in str(e_values):
+            error_title = "Unable to Add"
+            error_description = f"{e_values.args[1]}"
+        else:
+            error_title = "Error"
+            error_description = "Oops! Something went wrong.\nTry again!"
```

Comment:
Could we output `e_values` here as well?
We currently do not have logs available for debug, having e_values here can help to troubleshoot any other issue other than `404` or `Limit Exceeded`

Reply:
아아 동환이가 404랑 10명넘는거 외로는 그냥 general로 받으라고하긴했었는데
그리고 일단 embed로 보낼 info여서 specific info가 필요없긴할듯

Reply:
내가 생각한건, 
```
error_description = f"Oops! Something went wrong.Try again!\n{e_values.args[1]}"
```
이거였는데, so that we can see what unexpected error message is.
근데 이렇게 할시에 문제점은 customer is not suppose to see such error message. 그래서 굳이 안 고쳐도될듯.